doc/HD44780.rst

```diff
@@ -1,46 +1,52 @@
 HD44780
--------
+=======
+
+Introduction
+------------
+The HD44780 is sufficiently different from the other supported LCD displays to
+warrant a dedicated page to describe its features (and limitations).
 
 .. image:: images/hd44780_clock.jpg
 
-The HD44780 is a popular LCD display that is widely available and cheap.  It
-is available in several sizes with 16 character x 2 line and  20 character
-by 4 line displays the most common.  As with most LCD displays, they come with
-a backlight which luma.lcd can control either using PWM modulation or with a
-simple GPIO driven switch.
+It is a very popular LCD display that is widely available, cheap and
+available in several form factors with 16 character x 2 line and 20 character
+by 4 line displays the most common.  As with most LCD displays, it comes with
+a backlight which ``luma.lcd`` can control using a GPIO pin.  PWM modulation
+is supported if varying the intensity of the backlight is desired.
 
-They are normally connected to SBCs using a parallel ('bit bang') style interface
-but are also sold pre-connected to a few different I2C backpacks including the
-PCF8574, MCP23008 and MCP23017 expanders.
+They are normally connected to SBCs using a parallel 6800-series interface
+implemented bitbang-style using the SBCs GPIO pins.  Alternatively, they are
+also sold pre-connected to a few different I2C backpacks including the
+PCF8574, MCP23008 and MCP23017 I2C expanders.
 
 .. Note:
-  Currently luma.lcd only supports the parallel and pcf8574 interfaces
+  Currently ``luma.core`` only supports the ``bitbang_6800`` and ``pcf8574`` interfaces
```

Comment:
side note; do you have plans to work on these MCP23008 and MCP23017 expanders @dhrone? :)

Reply:
Probably.  I have ordered a device with an MCP23017 expander to play with but don't know when I'll get to it.  I looked through the spec sheets for them and they are a bit more complicated than the PCF8574 but don't seem like that big a challenge.  Strangely, when I searched i couldn't find a display pre-wired with a MCP23008. If you happen to have a pointer to one let me know.  Otherwise, I'll eventually order some MCP23008 chips and solder my own.